.github/workflows/ci.yaml

```diff
@@ -227,7 +224,7 @@ jobs:
 
   cargo-deny:
     needs: changed_files
-    if: ${{needs.changed_files.outputs.rs_changed == 'true' || needs.changed_files.outputs.ci_changed == 'true' || needs.changed_files.outputs.windows_changed == 'true'}}
+    if: ${{needs.changed_files.outputs.rs_changed == 'true' || needs.changed_files.outputs.ci_changed == 'true'}}
```

Comment:
After reintroducing the `windows_changed` output, we'd need to undo this change, to rerun cargo-deny also on window-specific changes.